foundry.toml

```diff
@@ -2,7 +2,7 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-solc = "0.8.18"
+solc = "0.8.23"
```

Comment:
Can you just add a small comment to the Readme that says something like "Compiler defaults to 8.23 but it can be adjusted in the foundry toml"